basic.obo

```diff
@@ -219494,39 +220777,189 @@ created_by: http://orcid.org/0000-0001-9897-3238
+name: metabolic zone of liver
+def: "Any of the metabolic zones found in the hepatic acinus. Liver tissue is heterogenous at the level of morphometry and histochemistry. This heterogeneity is linked to the position of a cell within the functional unit of the tissue, which, in turn, is related to the blood supply: Cells located in the upstream zone differ from those in the downstream zone with respect to subcellular structures, key enzymes, translocators, and receptors and, therefore, have different metabolic capacities. This is the basis for \"metabolic zonation\", a concept first proposed for carbohydrate metabolism and later expanded to include amino acid and ammonia metabolism, xenobiotic metabolism, protective metabolism, and formation of plasma proteins." [http://www.ncbi.nlm.nih.gov/pubmed/8839925]
+synonym: "hepatic acinus metabolic zone" EXACT [http://www.ncbi.nlm.nih.gov/pubmed/8839925]
+is_a: UBERON:0004119 ! endoderm-derived structure
+property_value: dc-creator http://orcid.org/0000-0003-2034-601X
+creation_date: 2020-09-29T15:45:36Z
+
+[Term]
+id: UBERON:8400006
+name: liver lobule periportal region
+def: "The region of the liver lobule that is closest to the portal triad." [http://www.ncbi.nlm.nih.gov/pubmed/28126520]
+synonym: "hepatic acinus zone 1" RELATED [http://www.ncbi.nlm.nih.gov/pubmed/28126520]
+is_a: UBERON:8400005 ! metabolic zone of liver
+property_value: dc-creator http://orcid.org/0000-0003-2034-601X
+relationship: part_of UBERON:0004647 ! liver lobule
+creation_date: 2020-10-05T12:56:57Z
+
+[Term]
+id: UBERON:8400007
+name: liver lobule centrilobular region
+def: "The region of the liver lobule that is closest to the central vein." [http://www.ncbi.nlm.nih.gov/pubmed/28126520]
+synonym: "hepatic acinus zone 3" RELATED [http://www.ncbi.nlm.nih.gov/pubmed/28126520]
+is_a: UBERON:8400005 ! metabolic zone of liver
+property_value: dc-creator http://orcid.org/0000-0003-2034-601X
+relationship: part_of UBERON:0004647 ! liver lobule
+creation_date: 2020-10-05T15:43:18Z
+
+[Term]
+id: UBERON:8400008
+name: liver lobule midzonal region
+def: "The region of the liver lobule that is between the periportal and centrilobular regions." [http://www.ncbi.nlm.nih.gov/pubmed/28126520]
+synonym: "hepatic acinus zone 2" RELATED [http://www.ncbi.nlm.nih.gov/pubmed/28126520]
+is_a: UBERON:8400005 ! metabolic zone of liver
+property_value: dc-creator http://orcid.org/0000-0003-2034-601X
+relationship: part_of UBERON:0004647 ! liver lobule
+creation_date: 2020-10-05T15:52:22Z
+
+[Term]
+id: UBERON:8410000
+name: duodeno-jejunal junction
+def: "The anatomical junction between the duodenum and the jejunum." [http://www.ncbi.nlm.nih.gov/pubmed/23961897, http://www.ncbi.nlm.nih.gov/pubmed/29064554]
+synonym: "duodeno-jejunal flexure" EXACT []
+synonym: "duodenojejunal flexure" EXACT []
+synonym: "duodenojejunal junction" EXACT []
+synonym: "flexura duodenojejunalis" EXACT [] {http://purl.obolibrary.org/obo/uberon/core#LATIN=""}
```

Comment:
@paolaroncaglia - thought we fixed this?  Don't want it to be a release blocker.  But note - could easily be fixed by hand in OBO if problems in Protege.

Reply:
@dosumis 
I thought we fixed that too, but it might have been another term. Hopefully addressed here (please double-check as I hand-edited OBO - to be faster - as my Protege settings are on EFO atm): https://github.com/obophenotype/uberon/pull/1792